js/beaxy.js

```diff
@@ -0,0 +1,414 @@
+        //        "weeklyWithdrawalThreshold":2.0E7,
+        //        "monthlyDepositThreshold":1.0E9,
+        //        "monthlyWithdrawalThreshold":5.0E7,
+        //        "dailyDepositLimit":1.0E9,
+        //        "dailyWithdrawalLimit":250000.0,
+        //        "weeklyDepositLimit":1.0E9,
+        //        "weeklyWithdrawalLimit":2.0E7,
+        //        "monthlyDepositLimit":1.0E9,
+        //        "monthlyWithdrawalLimit":5.0E7,
+        //        "minimalWithdrawal":875.0,
+        //        "type":"crypto"
+        //     }
+        // ]
+        //
+        const result = {};
+        for (let i = 0; i < response.length; i++) {
+            const currency = response[i];
+            const code = this.safeString (currency, 'currency');
+            const precision = this.safeInteger (currency, 'precision');
+            const name = this.safeString (currency, 'name');
+            result[code] = {
+                'id': code,
+                'code': code,
+                'name': name,
+                'active': true,
+                'fee': undefined,
+                'precision': precision,
+                'limits': {
+                    'amount': {
+                        'min': undefined,
+                        'max': undefined,
+                    },
+                    'price': {
+                        'min': undefined,
+                        'max': undefined,
+                    },
+                    'cost': {
+                        'min': undefined,
+                        'max': undefined,
+                    },
+                    'withdraw': {
+                        'min': undefined,
+                        'max': undefined,
+                    },
+                },
+            };
+        }
+        return result;
+    }
+
+    parseTrade (trade, market = undefined) {
+        const side = this.safeString (trade, 'side');
+        const timestamp = this.safeInteger (trade, 'timestamp');
+        const price = this.safeFloat (trade, 'price');
+        const amount = this.safeFloat (trade, 'size');
+        let cost = undefined;
+        if ((price !== undefined) && (amount !== undefined)) {
+            cost = price * amount;
+        }
+        let symbol = undefined;
+        if ((symbol === undefined) && (market !== undefined)) {
+            symbol = market['symbol'];
+        }
+        return {
+            'info': trade,
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'symbol': symbol,
+            'id': undefined,
+            'order': undefined,
+            'type': undefined,
+            'takerOrMaker': undefined,
+            'side': side.toLowerCase (),
+            'price': price,
+            'amount': amount,
+            'cost': cost,
+            'fee': undefined,
+        };
+    }
+
+    async fetchTrades (symbol, since = undefined, limit = undefined, params = {}) {
```

Comment:
We also need to make sure that `since` and `limit` are propagated in the request, if supported on the exchange side.